src/main/java/teammates/ui/webapi/output/FeedbackSessionData.java

```diff
@@ -214,12 +218,19 @@ public Long getDeletedAtTimestamp() {
      */
     public void hideInformationForStudent() {
         setResultVisibleFromTimestamp(null);
-        setGracePeriod(null);
         setSessionVisibleSetting(null);
         setCustomSessionVisibleTimestamp(null);
         setResponseVisibleSetting(null);
         setCustomResponseVisibleTimestamp(null);
+    }
+
+    /**
+     * Hides some attributes.
+     */
+    public void hideInformation() {
```

Comment:
The method name is vague. `hideInformation`: what kind of information? for what purpose? Although I admit I can't really suggest a better name without making it too long.

Reply:
It's only used on it's own once in `GetFeedbackSessionAction` for `INSTRUCTOR_RESULT` so I've changed it's name to reflect that appropriately